consensus/polybft/signer/utils.go

```diff
@@ -17,18 +17,18 @@ var (
 )
 
 const (
-	DomainValidatorSetString      = "DOMAIN_CHILD_VALIDATOR_SET"
-	DomainCheckpointManagerString = "DOMAIN_CHECKPOINT_MANAGER"
-	DomainCommonSigningString     = "DOMAIN_COMMON_SIGNING"
-	DomainStateReceiverString     = "DOMAIN_STATE_RECEIVER"
+	DomainValidatorSetString  = "DOMAIN_CHILD_VALIDATOR_SET"
+	DomainBlockMetaDataString = "DOMAIN_BRIDGE_META"
+	DomainCommonSigningString = "DOMAIN_COMMON_SIGNING"
+	DomainStateReceiverString = "DOMAIN_STATE_RECEIVER"
 )
 
 var (
 	// domain used to map hash to G1 used by (child) validator set
 	DomainValidatorSet = crypto.Keccak256([]byte(DomainValidatorSetString))
 
 	// domain used to map hash to G1 used by child checkpoint manager
-	DomainCheckpointManager = crypto.Keccak256([]byte(DomainCheckpointManagerString))
+	DomainBlockMeta = crypto.Keccak256([]byte(DomainBlockMetaDataString))
 
 	DomainCommonSigning = crypto.Keccak256([]byte(DomainCommonSigningString))
 	DomainStateReceiver = crypto.Keccak256([]byte(DomainStateReceiverString))
```

Comment:
While we are changing these domains, remove the `DomainStateReceiver` and its string, and wherever it is used, change it so that `DomainBridge` is used instead, because that is what new contracts expect.